src/main/java/io/anserini/document/CarDocument.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Anserini: An information retrieval toolkit built on Lucene
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.document;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+/**
+ * A paragraph object in the CAR dataset ver2.0. The paraID serves as the id.
+ * readNextRecord() is not used because CarCollection will load the .cbor file directly from the disk
+ * Reference: http://trec-car.cs.unh.edu/datareleases/
+ */
+public class CarDocument implements SourceDocument {
+  private final String paraID;
+  private final String paragraph;
+
+  public CarDocument(String paraID, String paragraph) {
+    this.paraID = paraID;
+    this.paragraph = paragraph;
+  }
+
+  /**
+   *
+   * @param bRdr
+   * @return
+   * @throws IOException
```

Comment:
I actually meant the javadoc of the `readNextRecord` function....There are two ways to do it:

1. To put comments in the front of the class (which you currently do) then I think it is probably better to just remove the javadoc of the `readNextRecord` function
2. Or you can move the comments in the javadoc of the `readNextRecord` function.

Personally, I prefer the second option but it is up to you.